keyvi/src/cpp/dictionary/dictionary_compiler.h

```diff
@@ -109,7 +109,9 @@ class DictionaryCompiler
       if (params_.count(TEMPORARY_PATH_KEY) == 0) {
         params_[TEMPORARY_PATH_KEY] =
             boost::filesystem::temp_directory_path().string();
-
+      } else {
```

Comment:
seems there is the same assignment on line: https://github.com/cliqz-oss/keyvi/pull/118/files#diff-a9c1c844ac65d3f1d74ecee3d93fe047L118